chainerx_build_helper.py

```diff
@@ -40,6 +40,13 @@ def run(self):
         if cmake_version < '3.1.0':
             raise RuntimeError('CMake >= 3.1.0 is required to build ChainerX')
 
+        self.use_ninja = False
+        try:
+            out = subprocess.check_output(['ninja', '--version'])
+            self.use_ninja = True
+        except OSError:
```

Comment:
I think it should be controlled by an environment variable for example, instead of existence of the command.
Maybe it's installed but the user don't want to use it.
Or it could be an unrelated command.

Reply:
Done. Maybe come up with a better name for it?